website/static/js/paginator.js

```diff
@@ -111,12 +111,16 @@ var Paginator = oop.defclass({
         }
     },
     nextPage: function(){
-        this.currentPage(this.currentPage() + 1);
-        this.fetchResults();
+        if (this.currentPage() !== this.numberOfPages() - 1) {
```

Comment:
The button is disabled. However, the issue is when you are on the second to the last page of logs. Click on the next page once, it sends a request to get the logs of last page. While the logs are loading, you are still on second to last page and you click on the next page twice in a roll, then it creates the problem.